mscore/qml/palettes/Palette.qml

```diff
@@ -387,36 +509,46 @@ GridView {
             }
 
             background: Rectangle {
-                id: cellBackground
-
-                color: globalStyle.voice1Color
-                opacity: 0.0
+                color: "transparent"
```

Comment:
@dmitrio95, fixed! I had to explicitly set the cell background color to "transparent" here, otherwise it was white by default hence the grid was no longer visible.